recipes/nest/activate.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+NEST_INSTALL_DIR=$PREFIX
+NEST_DATA_DIR="$NEST_INSTALL_DIR/share/nest"
+NEST_DOC_DIR="$NEST_INSTALL_DIR/share/doc/nest"
+NEST_MODULE_PATH="$NEST_INSTALL_DIR/lib/nest"
+NEST_PYTHON_PREFIX="$NEST_INSTALL_DIR/lib/python3.6/site-packages"
```

Comment:
The Python version dependence should be removed from here.
```suggestion
NEST_PYTHON_PREFIX="!!!SP_DIR!!!"
```
Then instead of copying the file add `sed` to the build dependencies and in the build script use:
```
sed "s#!!!SP_DIR!!!#${SP_DIR}#g" activate.sh > "${PREFIX}/etc/conda/activate.d/activate.sh"
```
([`$SP_DIR` is defined in the build environment](https://conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#env-vars))

Reply:
The `!!!SP_DIR!!!` is never replaced in the activation script with the actual path for this Python version (see above).

Reply:
Sorry, but there was a problem syncing my local branch. It's done.